gkr_iop/src/selector.rs

```diff
@@ -50,21 +53,29 @@ impl<E: ExtensionField> SelectorType<E> {
                 }
                 Some(sel.into_mle())
             }
-            SelectorType::KeccakRound(round, _, _expr) => {
-                // input(y) - pad = \sum_b ( sel(y; b[5..]) * (record(b) - 1) )
-                let eq = build_eq_x_r_vec(out_point);
-                let selected_id = CYCLIC_POW2_5[*round];
-                let sel: Vec<_> = iproduct!(0..32, eq.iter().enumerate())
-                    .par_bridge()
-                    .map(|(low_id, (hgh_id, s))| {
-                        if low_id == selected_id && hgh_id < num_instances {
-                            *s
-                        } else {
-                            E::ZERO
+            SelectorType::OrderedSparse32 { indices, .. } => {
```

Comment:
TODO: suggest from @dreamATD : we can split lookup into 32 evaluation, so can benefit from rotation reduced selector design (#layer - rotation_num_var)